autoload/lsp/internal/inlay_hints.vim

```diff
@@ -0,0 +1,95 @@
+let s:use_vim_textprops = lsp#utils#_has_textprops() && !has('nvim')
```

Comment:
given that virtual text was added later, we probably want to check the version too? Looking at the vim commit history, seems like there were several patches to make it work.

probably may even want to check the current latest version. [patch 9.0.0167](https://github.com/vim/vim/commit/38ea2733b4dd8e9bfc4a5bb5c11cb661bf000c4e)

https://github.com/prabirshrestha/vim-lsp/blob/309e9e5c8103bee69eabd152c09eaeec2e0a11f9/autoload/lsp/utils.vim#L21-L25